Packs/McAfee_ESM_v2/Integrations/McAfee_ESM_v2/McAfee_ESM_v2.py

```diff
@@ -0,0 +1,811 @@
+from typing import Dict, List, Tuple, Optional, Callable
+from urllib3 import disable_warnings
+from CommonServerPython import *
+import demistomock as demisto
+from datetime import timedelta, datetime
+import time
+
+
+disable_warnings()
+CONTEXT_INTEGRATION_NAME = 'McAfeeESM.'
+
+
+class McAfeeESMClient(BaseClient):
+
+    demisto_format = '%Y-%m-%dT%H:%M:%SZ'
+
+    def __init__(self, params: Dict):
+        self.args = demisto.args()
+        self.__user_name = params.get('credentials', {}).get('identifier', '')
+        self.__password = params.get('credentials', {}).get('password', '')
+        self.difference = int(params.get('timezone', 0))
+        self.version = params.get('version', '10.2')
+        super(McAfeeESMClient, self).__init__(
+            'https://' + params.get('ip', '') + ':' + params.get('port', '443') + f'/rs/esm/v2/',
+            proxy=params.get('proxy', False),
+            verify=not params.get('insecure', False)
+        )
+        self._headers = {'Content-Type': 'application/json'}
+        self.__login()
+        self.__cache: Dict = {
+            'users': [],
+            'org': [],
+            'status': []
+        }
+
+    def __del__(self):
+        self.__logout()
+
+    def _is_status_code_valid(self, *other):    # noqa
```

Comment:
Do this change and then you can get rid of the whitelist.py of vulture.
```suggestion
    def _is_status_code_valid(self, *_other):    # noqa
```